app-text/diffpdf/files/diffpdf-2.1.3-qt6.patch

```diff
@@ -0,0 +1,407 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+new file mode 100644
+index 0000000..c7e6d76
+--- /dev/null
++++ b/CMakeLists.txt
+@@ -0,0 +1,25 @@
++cmake_minimum_required(VERSION 3.28)
++project(diffpdf CXX)
++
++find_package(Qt6 REQUIRED COMPONENTS LinguistTools PrintSupport Widgets)
++find_package(PkgConfig REQUIRED)
++pkg_search_module(PopplerQt6 REQUIRED IMPORTED_TARGET poppler-qt6>=24.11.0)
++
++qt_standard_project_setup(REQUIRES 6.8 I18N_TRANSLATED_LANGUAGES cz de es fr)
```

Comment:
6.8 is probably not necessary here. I guess 6.5 would be completely fine and may enable other distros to easily consider your patch as well.

Reply:
Changed to 6.4.2. Tested to compile against that on Debian 12.